AutocompleteClient/FW/UI/DefaultSearchItemCell.xib

```diff
@@ -35,7 +35,7 @@
             <connections>
                 <outlet property="labelText" destination="swl-kL-qeP" id="B4N-VJ-CRQ"/>
             </connections>
-            <point key="canvasLocation" x="-49" y="104"/>
+            <point key="canvasLocation" x="-49.600000000000001" y="103.44827586206897"/>
```

Comment:
should these x/y coordinates be this specific?

Reply:
It's just the position of the top view in the canvas. It's pretty irrelevant.